src/components/atoms/ImagePicker/Component.tsx

```diff
@@ -263,8 +246,18 @@ const ImagePicker = forwardRef<View, ImagePickerProps>(
         ),
       [options]
     );
+    const getPickerData = () => {
+      switch (launcherType) {
+        case 'camera':
+          return [CAMERA_KIND];
+        case 'image-library':
```

Comment:
Can we make these values to Enum?

Reply:
Thank you for the suggestion, I've changed it to use enum instead of union type.